src/main/java/com/jlr/ttl/ds/api/dto/entity/Location.java

```diff
@@ -0,0 +1,30 @@
+package com.jlr.ttl.ds.api.dto.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "locations")
+public class Location {
+    @Id
+    @Column(name="id")
+    private String lId;
+    @Column(name = "description")
+    private String lDescription;
+
+    public Location(String lId, String lDescription) {
```

Comment:
To reduce boilerplate code, use the NoArgsConstructor, AllArgsContructor, Data, and Builder annotattions. These generate the constructors getters and setters for you